packages/fluentui/accessibility/src/behaviors/index.ts

```diff
@@ -109,3 +109,4 @@ export { default as cardBehavior } from './Card/cardBehavior';
 export * from './Card/cardBehavior';
 export { default as cardFocusableBehavior } from './Card/cardFocusableBehavior';
 export { default as cardChildrenFocusableBehavior } from './Card/cardChildrenFocusableBehavior';
+export { default as videoBehavior } from './Video/videoBehavior';
```

Comment:
Please don't forget to add new behavior to packages/fluentui/accessibility/test/behaviors/behavior-test.tsx

Reply:
Done